src/main/java/org/spongepowered/common/world/server/SpongeWorldTypeTemplate.java

```diff
@@ -78,6 +98,8 @@
         this.logicalHeight = builder.logicalHeight;
         this.maximumHeight = builder.maximumHeight;
         this.coordinateScale = builder.coordinateMultiplier;
+
+        // Sponge
         this.createDragonFight = builder.createDragonFight;
```

Comment:
This isn't needed anymore, I think. Vanilla has this on the builder. Fairly sure it is in their codec unless you found otherwise?